script.js

```diff
@@ -1,16 +1,105 @@
 // Write your JavaScript code here!
+window.addEventListener("load", function(){
+    let listedPlanets;
+    // Set listedPlanetsResponse equal to the value returned by calling myFetch()
+    listedPlanets = "https://handlers.education.launchcode.org/static/planets.json";
 
-window.addEventListener("load", function() {
-
-   let listedPlanets;
-   // Set listedPlanetsResponse equal to the value returned by calling myFetch()
-   let listedPlanetsResponse;
-   listedPlanetsResponse.then(function (result) {
-       listedPlanets = result;
-       console.log(listedPlanets);
-   }).then(function () {
-       console.log(listedPlanets);
-       // Below this comment call the appropriate helper functions to pick a planet fom the list of planets and add that information to your destination.
-   })
-   
-});
\ No newline at end of file
+    // Fetching Planetary Data
+    this.fetch(listedPlanets).then(function(result) {
+        //listedPlanets=results;
+        result.json().then(function(json) {
+            let listedPlanetsResponse = document.getElementById("missionTarget");
+            //console.log(listedPlanets);
+            let missionDestination = Math.floor(Math.random()*json.length -1);
+           listedPlanetsResponse.innerHTML = `
+           <h1>Mission Destination</h1>
+           <ol>
+              <li>Name: ${json[missionDestination].name}</li>
+              <li>Diameter: ${json[missionDestination].diameter}</li>
+              <li>Star: ${json[missionDestination].star}</li>
+              <li>Distance from Earth: ${json[missionDestination].distance}</li>
+              <li>Number of Moons: ${json[missionDestination].moons}</li>
+           </ol>
+           <img src="${json[missionDestination].image}"> `;
+        });
+     });
+
+
+//Submitting pilot, co-pilot, fuel level, & cargo mass
+let form = document.querySelector("form");
+form.addEventListener("submit", function(event){
+    let pilotName = document.querySelector("input[name=pilotName]");
```

Comment:
```suggestion
form.addEventListener("submit", function(event){
    event.preventDefault();
    let pilotName = document.querySelector("input[name=pilotName]");
```
I think some of the unit tests are not passing because the page refreshes after a successful form submission occurs. I'd recommend removing your `event.preventDefault()` from line 59, 72, and 84, and moving it above line 31 (keeping it inside your submit event listener). This will prevent the page from reloading after successful form submission, and keep your `faultyItems` list displayed on the page when the inputs are not empty or have invalid data types (i.e. pilot names are strings; fuel and cargo levels are numbers) and when rendering the status of all of the launch data.